client/src/app.jsx

```diff
@@ -17,13 +18,30 @@ class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      images: props.images
+      images: []
     };
+   // this.getData.bind(this);
+    this.getData();
   }
 
+  getData() {
+    let url = 'http://localhost:3003/house_images?id='+this.props.id;
 
-  render() {
+    console.log("Url is:", url);
+
+  fetch(url)
```

Comment:
Is this still working even though you commented out the fetch import statement?

Reply:
Yes. I didn't need to include the Fetch API as fetch is already a part of React